spec.bs

```diff
@@ -2741,6 +2777,59 @@ CORP violation report=] algorithm, as leaving it unfenced may cause a privacy le
   [=automatic beacon data/once=] set to true.
 </div>
 
+<div algorithm=get-initiator-ancestor-root-origin>
+  To <dfn>get the initiator ancestor root origin</dfn> of a given {{Document}} |sourceDocument|:
+
+  1. Let |navigable| be |sourceDocument|'s [=node navigable=].
+
+  1. While |navigable| is not null and |navigable| is not a [=fenced navigable container/fenced
+     navigable=]:
+  
+     1. Set |navigable| to |navigable|'s [=navigable/parent=].
+
+  1. If |navigable| is null, return null.
+
+  1. Otherwise:
```

Comment:
I don't think this is needed? We can just de-dent all of the below steps and put them right below without an "Otherwise", since the prior steps returns. Right?

Reply:
We can do away with this. I'm going to keep the other "otherwise" on the last step for readability (since it doesn't result in any indentations). Let me know if you disagree with that decision and I can remove that as well.